doc/page-features.h

```diff
@@ -151,4 +151,22 @@ foo.userData<Pod>().i;  // 42
 An interface implementation can expose a structure containing some user data. This user data is
 preserved when the structure is copied. Thus it can be made available in a different interface.
 
+\section ServerImplementation   Implementation of Server inside C++ Implementation of Interface 
+
+In order to skip using IPC.enabled:true inside qml implementation , and create server inside C++ delegate, one
```

Comment:
Hmm, can you re-phrase? Be more verbose? "In order to skip..." - that's just one consequence, when the server is registered in the C++ implementation. "inside qml implementation" is misleading, it is in the QML UI code.
At least: \c IPC.enabled: \c true,  " , " -> ", "

Reply:
updated